bin/argo-api-authn-scripts/ams-create-users-cloud-info.py

```diff
@@ -0,0 +1,420 @@
+                "/topics/" + topic_name + "?key=" + ams_token, verify=verify)
+            if topic_crt_req.status_code != 200:
+                LOGGER.critical(
+                    "Something went wrong while creating topic " +
+                    topic_name + "\nResponse: " + topic_crt_req.text)
+                continue
+
+            # modify the authorized users
+            modify_topic_req = requests.post(
+                "https://" + ams_host + "/v1/projects/" + ams_project +
+                "/topics/" + topic_name + ":modifyAcl?key=" + ams_token,
+                data=json.dumps({'authorized_users': [user_binding_name]}),
+                verify=verify)
+            LOGGER.critical(
+                "Modified ACL for topic: {} with users {}. " +
+                "Response from AMS {}".
+                format(
+                    topic_name, str(user_binding_name), modify_topic_req.text))
+
+        # modify the acl for each topic , to add all associated users
+        authorized_users = services[srv_type]
+        if len(authorized_users) != 0:
+
+            get_topic_acl_req = requests.get(
+                "https://" + ams_host + "/v1/projects/" + ams_project +
+                "/topics/" + srv_type + ":acl?key=" + ams_token, verify=verify)
+
+            if get_topic_acl_req.status_code == 200:
+                acl_users = json.loads(get_topic_acl_req.text)
+                authorized_users = \
+                    authorized_users + acl_users["authorized_users"]
+                modify_topic_acl_req = requests.post(
+                    "https://" + ams_host + "/v1/projects/" + ams_project +
+                    "/topics/" + srv_type + ":modifyAcl?key=" + ams_token,
+                    data=json.dumps({'authorized_users': authorized_users}),
+                    verify=verify)
+                LOGGER.critical(
+                    "Modified ACL for topic: {} with users {}. " +
+                    "Response from AMS {}".format(
+                        srv_type, str(authorized_users),
+                        modify_topic_acl_req.text))
+            else:
+                LOGGER.critical(
+                    "Couldn't get ACL for topic {}. Response from AMS {}".
+                    format(srv_type, get_topic_acl_req.text))
+
+        LOGGER.critical("Service Type: " + srv_type)
+        LOGGER.critical("Missing DNS: " + str(missing_dns))
+        LOGGER.critical("Total Users Created: " + str(user_count))
+        LOGGER.critical("-----------------------------------------")
+
+
+def main(args=None):
+
+    # set up the config parser
+    config = ConfigParser.ConfigParser()
+
+    # check if config file has been given as cli argument else
+    # check if config file resides in /etc/argo-api-authn/ folder else
+    # check if config file resides in local folder
+    if args.ConfigPath is None:
+        cfg_file = "/etc/argo-api-authn/conf.d/ams-create-users-gocdb.cfg"
```

Comment:
will this script use the same configuration file as ams-create-users-gocdb?

Reply:
Changed config file to use a different one, ams-create-users-cloud-info.cfg